stats/lib/platform-stats-fetchers.js

```diff
@@ -68,13 +68,27 @@ export const fetchParticipantsWithTopMeasurements = async (pgPool, filter) => {
  */
 export const fetchDailyRewardTransfers = async (pgPool, filter) => {
   const { rows } = await pgPool.query(`
-    SELECT day::TEXT, SUM(amount) as amount
+    SELECT day::TEXT, to_address, amount
     FROM daily_reward_transfers
     WHERE day >= $1 AND day <= $2
-    GROUP BY day
-    ORDER BY day
   `, [filter.from, filter.to])
-  return rows
+  const days = {}
+  for (const row of rows) {
+    if (!days[row.day]) {
+      days[row.day] = {
+        day: row.day,
+        amount: '0',
+        transfers: []
+      }
+    }
+    const day = days[row.day]
+    day.amount = String(BigInt(day.amount) + BigInt(row.amount))
```

Comment:
Do you have any concerns about the performance impact of repeatedly converting `day.amount` between BigInt and String?

Reply:
Nope, I noticed this redundant conversion too, but kept it for simplicity.

Here's a small benchmark:

```js
> ;{ const start = new Date(); for (let i = 0; i < 1e6; i++) String(BigInt('13')); console.log(new Date() - start); }
40
```

It takes 40ms on my laptop to perform this operation one million times.

Reply:
> Nope, I noticed this redundant conversion too, but kept it for simplicity.
> 
> Here's a small benchmark:
> 
> ```js
> > ;{ const start = new Date(); for (let i = 0; i < 1e6; i++) String(BigInt('13')); console.log(new Date() - start); }
> 40
> ```
> 
> It takes 40ms on my laptop to perform this operation one million times.

I think v8 is going to optimise your inner loop into a no-op.

I wrote a benchmark that's IMO better, but it still took only like 125ms to complete, so I agree with your conclusion.

```js
const start = new Date();                                                       
let sum = '0'                                                                   
for (let i = 0n; i < 10n ** 6n; i++) {                                          
  sum = String(BigInt(sum) + i)                                                 
}                                                                               
console.log('Duration:', new Date() - start);                                   
console.log('Sum:', sum)   
```